mne/channels/channels.py

```diff
@@ -1600,6 +1600,14 @@ def find_ch_adjacency(info, ch_type):
     you want to load a template for a given montage use
     :func:`read_ch_adjacency` directly.
 
+    .. warning::
+        If Delaunay triangulation is used to calculate the adjacency matrix it
+        may yield partially unexpected results (e.g., include unwanted edges
+        between non-adjacent sensors). Therefore, it is recommended to check
+        (and, if necessary, manually modify) the result by inspecting it
+        via :func:`mne.viz.plot_ch_adjacency`.
+        Delete this line @eort...
```

Comment:
@eort can you reply to this comment on GH using the document plus/minus suggest-change feature suggesting to delete this line? Then I can commit it, and you should get co-author credit for this change

It's a hack but it was the easiest way I saw to get you credit in the commit history

Reply:
FYI you can also manually type
```
Co-authored-by: Name Surname <email@addre.ss>
```
on line 3 of a multi-line commit message. But you have to get the syntax just right and use a name/email just as they're spelled in our mailmap.  So it's not actually easier :)

Reply:
```suggestion
```
Like this? 
thanks for your efforts. 🙏Would not have been necessary.

Reply:
Perfect, thanks :+1:

Reply:
> FYI you can also manually type
> 
> ```
> Co-authored-by: Name Surname <email@addre.ss>
> ```
> 
> on line 3 of a multi-line commit message. But you have to get the syntax just right and use a name/email just as they're spelled in our mailmap. So it's not actually easier :)

I love this comment on multiple levels. 😂